dotcom-rendering/src/components/ImageComponent.tsx

```diff
@@ -321,16 +328,36 @@ export const ImageComponent = ({
 					position: relative;
 				`}
 			>
-				<Picture
-					role={role}
-					format={format}
-					master={image.url}
-					alt={element.data.alt ?? ''}
-					width={imageWidth}
-					height={imageHeight}
-					isLazy={!isMainMedia}
-					isMainMedia={isMainMedia}
-				/>
+				{renderingTarget === 'Apps' ? (
+					<Island>
```

Comment:
hmm yes, that's an interesting point. Ideally we would still render the image markup on the server and just hydrate for interactivity on the client. I _think_ that, because we access `window.*` from inside a click listener, this shouldn't cause a problem server-side. Additionally (but not water tight evidence!) I haven't seen any errors in the server console about this.